Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Wrangling"
 uuid = "9e19721c-8a1a-4e7b-ae00-562464868f98"
 authors = ["Invenia Technical Computing Corporation"]
-version = "0.1.2"
+version = "0.2.0"
```

Comment:
Why is it a breaking change?

Reply:
Aren't dep upgrades minor? Or only if it's 1.0?

Reply:
Ya minor for 1.0, patch for pre-1.0 https://github.com/SciML/ColPrac#changing-dependency-compatibility